pkg/image/docker_client.go

```diff
@@ -109,6 +109,14 @@ func (i DockerClient) DeleteImages(images []string, retries int) []error {
 	return errs
 }
 
+func (i DockerClient) RunImage(image string, args ...string) ([]string, error) {
```

Comment:
I don't see where this is used in this PR.

Reply:
`cmd/sonobuoy/app/images.go` under `collectPluginImages`

https://github.com/vmware-tanzu/sonobuoy/pull/1192/files#diff-7adc43f24646df465c7cf42f8867fd929edc34efe0a346af720bc909766e2114R418